src/main/java/uk/gov/hmcts/divorce/citizen/event/CitizenInviteApplicant2.java

```diff
@@ -53,6 +58,8 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
 
         // TODO - send email to applicant 2 (to be done in NFDIV-689)
```

Comment:
can remove this comment now ?

Reply:
this is the email notification for applicant 1
notification for applicant 2 is still to be done - https://tools.hmcts.net/jira/browse/NFDIV-689